src/renderer/dx/DxRenderer.hpp

```diff
@@ -178,12 +180,18 @@ namespace Microsoft::Console::Render
         bool _haveDeviceResources;
         ::Microsoft::WRL::ComPtr<ID3D11Device> _d3dDevice;
         ::Microsoft::WRL::ComPtr<ID3D11DeviceContext> _d3dDeviceContext;
-        ::Microsoft::WRL::ComPtr<IDXGIFactory2> _dxgiFactory2;
-        ::Microsoft::WRL::ComPtr<IDXGISurface> _dxgiSurface;
-        ::Microsoft::WRL::ComPtr<ID2D1RenderTarget> _d2dRenderTarget;
+
+        ::Microsoft::WRL::ComPtr<ID2D1Device> _d2dDevice;
+        ::Microsoft::WRL::ComPtr<ID2D1DeviceContext> _d2dDeviceContext;
+        ::Microsoft::WRL::ComPtr<ID2D1Bitmap1> _d2dBitmap;
         ::Microsoft::WRL::ComPtr<ID2D1SolidColorBrush> _d2dBrushForeground;
         ::Microsoft::WRL::ComPtr<ID2D1SolidColorBrush> _d2dBrushBackground;
-        UINT _swapChainFlags;
+
+        ::Microsoft::WRL::ComPtr<IDXGIFactory2> _dxgiFactory2;
+        ::Microsoft::WRL::ComPtr<IDXGIDevice> _dxgiDevice;
+        ::Microsoft::WRL::ComPtr<IDXGISurface> _dxgiSurface;
+
+        DXGI_SWAP_CHAIN_DESC1 _swapChainDesc;

```

Comment:
unless we're *certain* we're setting all its fields, we gotta zero-initialize this either here or in the ctor.

Reply:
done.